lib/workload/stateless/statelessStackCollectionClass.ts

```diff
@@ -14,12 +14,14 @@ import {
   SequenceRunManagerStack,
   SequenceRunManagerStackProps,
 } from './stacks/sequence-run-manager/deploy/component';
+import { BsRunsUploadManagerStackProps } from './bs_runs_upload_manager/deploy/lib/stacks/bs_runs_upload_manager_stack';
```

Comment:
Need fix mid-path:

```
import { BsRunsUploadManagerStackProps } from './stacks/bs-runs-upload-manager/deploy/lib/stacks/bs_runs_upload_manager_stack';
```

Reply:
Fixed in b4041ad7905c37aaccd6af7fb0f9180cb1910213